custom_components/oasis_mini/pyoasiscontrol/device.py

```
@@ -0,0 +1,762 @@
+                "brightness_max": _parse_int(values[13]),
+                "wifi_connected": _bit_to_bool(values[14]),
+                "repeat_playlist": _bit_to_bool(values[15]),
+                "autoplay": _parse_int(values[16]),
+                "auto_clean": _bit_to_bool(values[17]),
+            }
+
+            # Optional trailing field(s)
+            if n > 18:
+                status["software_version"] = values[18]
+
+        except Exception:  # noqa: BLE001
```

Ruff (0.14.5)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)